src/_includes/prerelease.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  <p><strong>Warning</strong>: This API is currently in pre-release status and is only available to approved early access partners. The API is under development and might change before being generally released. To become an early access partner, contact your SAP Concur Representative.</p>
```

Comment:
As discussed, let's change this from Warning to Important.

Reply:
Changed.